src/child/containers/sidebars/star/star-compact.less

```diff
@@ -0,0 +1,9 @@
+.compact {
```

Comment:
I don't think `star` makes sense much now, I understand it was there in the Angular version, but as we don't have a star component/container, I think it makes sense to move this to be with the relevant component.

How about rename `star-compact.less` => `unfavourite-confirmation-compact.less` and `star.less` => `unfavourite-confirmation.less`?

They could all be in the same dir in components, i.e.:

```
child
    - components
        - favourite
             - Favourite.js
             - UnfavouriteConfirmation.js
             - unfavourite-confirmation-compact.less
             - unfavourite-confirmation.less
```